worker/servicefactory/interface.go

```diff
@@ -22,6 +24,8 @@ type ControllerServiceFactory interface {
 	ControllerNode() *controllernodeservice.Service
 	// ModelManager returns the model manager service.
 	ModelManager() *modelmanagerservice.Service
+	// ModelServiceForUUID returns the model service for the provided model UUID.
+	ModelServiceForUUID(uuid model.UUID) *modelservice.Service
```

Comment:
Ah. Hmmm. I need to think about this one.

Reply:
This is gone.